src/test/java/integration/ios/driverproviders/IosDriverProvider.java

```diff
@@ -40,7 +40,7 @@ public WebDriver createDriver(@Nonnull Capabilities capabilities) {
   protected XCUITestOptions getXcuiTestOptions() {
     XCUITestOptions options = new XCUITestOptions();
     // on GitHub actions, first test run maybe extremely slow
-    options.setWdaLaunchTimeout(Duration.ofMinutes(10));
+    options.setWdaLaunchTimeout(Duration.ofSeconds(30));
```

Comment:
The tests have failed due to this change. There are corresponding comments above.